queries.yaml

```diff
@@ -1278,6 +1276,22 @@ post_process:
         (kind == 'major_road' and zoom < 13) or
         (kind == 'minor_road' and zoom < 15)
 
+  # Because we have is_bus_route boolean, we don't need the
+  # details about bus network, operator, and shield text until
+  # later zooms when there is enough space to label them
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 17
+      properties:
+        - bus_network
```

Comment:
Test with https://www.openstreetmap.org/way/190333972 which includes bus relations, visible in 16-17756-27464 and it's parent 15-8878-13732 tiles at 512px tile coordinates.